src/AutoSplitImage.py

```diff
@@ -98,6 +121,39 @@ def __init__(self, path: str):
         else:
             self.image_type = ImageType.SPLIT
 
+    def __parse_text_file(self, path: str):
+        if not TESSERACT_PATH:
+            error_messages.tesseract_missing(path)
+            return
+
+        with open(path, encoding="utf-8") as f:
+            data = toml.load(f)
+
+        self.texts = [text.lower().strip() for text in data["texts"]]
+        self.__rect = [
+            data["top_left"][0],
+            data["bottom_right"][0],
+            data["top_left"][1],
+            data["bottom_right"][1],
+        ]
+        self.__ocr_comparison_methods = data.get("methods", [0])
+        self.__fps_limit = data.get("fps_limit", 0)
+
+        if self.__validate_ocr():
+            error_messages.wrong_ocr_values(path)
+            return
+
+    def __validate_ocr(self):
+        values = self.__rect + self.__ocr_comparison_methods
+        values.append(self.__fps_limit)
+        return (
+            any(  # Check for invalid negative values
+                value < 0 for value in values
+            )
+            or self.__rect[1] <= self.__rect[0]
+            or self.__rect[3] <= self.__rect[2]
```

Comment:
I feel it would make more sense for a guard to return `True` when the validation passes.
Also slight optimization for the creation of `values`
```suggestion
        if not self.__validate_ocr():
            error_messages.wrong_ocr_values(path)
            return

    def __validate_ocr(self):
        values = [*self.__rect, *self.__ocr_comparison_methods, self.__fps_limit]
        return (
            all(  # Check for invalid negative values
                value >= 0 for value in values
            )
            and self.__rect[1] > self.__rect[0]
            and self.__rect[3] > self.__rect[2]
```